### ISSUE TITLE
Can't Open Devil May Cry 4 (Non-3D)

### ISSUE THREAD

**Issue by Indra2108** [nikita36078/J2ME-Loader#494]
**Emulator version:**
1.4.3

**Game version:**
1.0.0

**Game resolution:**
240x320

**Device:**
Smartfren Andromax G

**Android version:**
4.1.2

**Description of the issue:**
Cannot open the game even though it uses jad files

Link Game : http://phoneky.com/games/?id=j4j30364

**Comment by amarcioz** [comment]
I tested this game is worked perfectly.
Samsung j1 2016
Android 5.1

**Comment by nikita36078** [comment]
Looks like another Android 4.x issue, I'm not sure if I can fix it.

**Comment by BinariGuy** [comment]
Devil May Cry Non 3D works very well on my Android
M4 SS4452
Android 5.1.1 Lollipop

**Comment by Indra2108** [comment]
Yes, I decided to try it too in KEmulator, and it works very well too. Looks like my phone has a problem.

**Comment by Bakugan77** [comment]
No way!! So cheap smartphone..But at least Realme 3 or redmi note 7 dude..Such an embarrassing  phone u use in 2019 Hahahh..I am gonna die laugghing😂😂😂
